content/en/integrations/kubernetes_state_core.md

```diff
@@ -0,0 +1,478 @@
+---
+title: Kubernetes State Metrics Core
+name: kubernetes_state_core
+kind: integration
+description: "Monitor the health of the workload running on your Kubernetes cluster. Track the status of your Kubernetes objects, correlate your microservice metrics, and more."
+short_description: "Track the status of your Kubernetes objects, correlate your microservice metrics, and more."
+categories:
+    - cloud
+    - configuration & deployment
+    - containers
+    - orchestration
+doc_link: /integrations/kubernetes_state_core/
+dependencies:
+    ['https://github.com/DataDog/documentation/blob/master/content/en/integrations/kubernetes_state_core.md']
+has_logo: true
+integration_title: Kubernetes State Metrics Core
+is_public: true
+public_title: Datadog-Kubernetes State Metrics Core Integration
+integration_id: "kubernetes_state_core"
+---
+
+## Overview
+
+Get metrics from Kubernetes service in real time to:
+
+- Visualize and monitor kubernetes states
+- Be notified about kubernetes failovers and events.
+
+The Kubernetes State Metrics Core check leverages [kube-state-metrics version 2+][1] and includes major performance and tagging improvments compared to the legacy `kubernetes_state` check.
+
+As opposed to the legacy check, with the Kubernetes State Metrics Core check you will not need to deploy `kube-state-metrics` in your cluster anymore.
+
+Kubernetes State Metrics Core provides a better alternative to the legacy `kubernetes_state` check as it offers more granular metrics and tags. See the [Major Changes][2] and [Data Collected][3] for more details.
+
+## Setup
+
+### Installation
+
+The Kubernetes State Metrics Core check is included in the [Datadog Cluster Agent][4] image, so you don't need to install anything else on your Kubernetes servers.
+
+### Requirements
+
+- Datadog Cluster Agent v1.12+ 
+
+### Configuration
+
+{{< tabs >}}
+{{% tab "Helm" %}}
+
+In your Helm `values.yaml`, add the following:
+
+```
+...
+datadog:
+...
+  kubeStateMetricsCore:
+    enabled: true
+...
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+## Migration from kubernetes_state to kubernetes_state_core
+
+### Backward incompatibility changes
+
+The Kubernetes State Metrics Core check is not backward compatible, please read the changes carefully before migrating from the legacy `kubernetes_state` check.
+
+`kubernetes_state.node.by_condition`
+: A new metric with node name granularity. It replaces `kubernetes_state.nodes.by_condition`.
+
+`kubernetes_state.persistentvolume.by_phase`
+: A new metric with persistentvolume name granularity. It replaces `kubernetes_state.persistentvolumes.by_phase`.
+
+`kubernetes_state.pod.status_phase`
+: The metric is tagged with pod level tags now, like `pod_name`.
+
+
+{{< tabs >}}
+{{% tab "Helm" %}}
+
+Enabling `kubeStateMetricsCore` in your Helm `values.yaml` will also configure the Agent to ignore the auto configuration file for legacy `kubernetes_state` check. The goal is to avoid running both checks simultaneously.
```

Comment:
```suggestion
Enabling `kubeStateMetricsCore` in your Helm `values.yaml` configures the Agent to ignore the auto-configuration file for legacy `kubernetes_state` check. The goal is to avoid running both checks simultaneously.
```